vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -159,9 +159,8 @@ vic_cesm_put_data()
 
         // evaporation, VIC: mm, CESM: kg m-2 s-1
         // TO-DO should we incorporate bare soil evap?
```

Comment:
It's not still relevant. Since we're using `evap` from `OUT_DATA` here, it's already incorporating soil evap.